src/components/SemesterView.vue

```diff
@@ -158,21 +169,75 @@ export default {
 .semesterView {
   display: flex;
   flex-direction: column;
-  margin-bottom: 40px;
+  margin: 1.5rem 3rem 3rem;
+  max-width: 49rem;
```

Comment:
Can we remove this to make it adjustable for bigger screen sizes? We also want to display four semesters in compact view for at least a 13-inch MacBook